src/components/common/YourCompany/CompanyName/CorrectNameOptions.vue

```diff
@@ -82,14 +82,15 @@ export default class CorrectNameOptions extends Vue {
   @Prop() readonly correctionNameChoices: Array<string>
 
   // local properties
-  private displayedOptions: Array<CorrectNameOptionIF> = []
-  private panel: number = null
-  private formType: CorrectionTypes = null
-  private currentFormType: CorrectionTypes = null
-  private isLoading = false
-  private isFormValid = false
-  private validateNameChange = false
-  private correctionNameOptions: Array<CorrectNameOptionIF> = [
+  protected displayedOptions: Array<CorrectNameOptionIF> = []
+  protected panel: number = null
+  protected formType: CorrectionTypes = null
+  protected currentFormType: CorrectionTypes = null
+  protected isLoading = false
+  protected isFormValid = false
+  protected validateNameChange = false
```

Comment:
These class variables should not be accessed externally, but we do want the template (HTML) and any sub-classes to see them.